src/test/with-feature-toggles.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { describe } from 'riteway';
+import dom from 'cheerio';
+import { withFeatureToggles } from '../with-feature-toggles';
+
+import { Consumer } from '../context';
+
+const render = ReactDOMServer.renderToStaticMarkup;
+
+const createTestHOCComponent = componentName => ({ propCheck } = {}) => (
```

Comment:
Is a dynamic component name necessary for testing withFeatureToggles? Think about what we are actually testing about withFeatureToggles.